core/elements/ons-switch.es6

```diff
@@ -45,10 +45,31 @@ limitations under the License.
       this._compile();
       ModifierUtil.initModifier(this, scheme);
 
+      this._addProperties();
       this._updateForCheckedAttribute();
       this._updateForDisabledAttribute();
     }
 
+    _addProperties() {
```

Comment:
Why do you use Object.defineProperty()? You should use ES6's setter and getter method syntax.